### ISSUE TITLE
Turtle CLI fails to build Android project using SDK 31

### ISSUE THREAD

**Issue by fwal** [expo/expo#2715]
### Environment

OS: alpine 3.8.1
Node: 8.11.4
Expo CLI: 2.4.0
Turtle CLI: 0.3.3
Expo SDK: 31.0.5

### Steps to Reproduce

1. `expo init`
2. (Set slug to previously released project)
3. `turtle build:android [...]`

### Expected Behavior

Turtle builds the application successfully.

### Actual Behavior

Turtle fails with the following errors (seems to be same as reported in #2502):
```sh
ERROR: Error: spawn ../../tools-public/check-dynamic-macros-android.sh ENOENT
    at _errnoException (util.js:992:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
    at onErrorNT (internal/child_process.js:372:16)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
ERROR: Failed to build standalone app: spawn ../../tools-public/check-dynamic-macros-android.sh ENOENT
ERROR: Error: spawn ../../tools-public/check-dynamic-macros-android.sh ENOENT
    at _errnoException (util.js:992:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
    at onErrorNT (internal/child_process.js:372:16)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
```

**Comment by dsokal** [comment]
I've already talked to @fwal on Expo Developers Slack. It looks like he upgraded `turtle-cli` version and he was caching the `.turtle/workingdir/android/sdk31` directory. We changed the shell app for Android SDK 31 in the meantime. The previous version didn't have `tools-public/check-dynamic-macros-android.sh` file. Removing the directory and running `turtle setup:android --sdk-version 31.0.0` should resolve the issue.

**Comment by fwal** [comment]
Turns out that this issue was due to the script using `bash` which isn't included in alpine. In other words: It didn't complain that it couldn't find the file, it couldn't find the shell 🙂

**Comment by dsokal** [comment]
@fwal I've discussed this issue with the Expo team and I'm afraid I have to ask you to just install bash on your alpine image. Using bash is in alignment with our natural workflow. 

Also, I just published `turtle-cli@0.3.5` and it now checks if `bash` is installed during the setup phase and it fails as soon as it discovers it's not in your `$PATH`.

We also updated our docs on `turtle-cli` and put `bash` in the Prerequisites: https://docs.expo.io/versions/v31.0.0/distribution/turtle-cli#prerequisites

I'm closing this issue.
